gluster_exporter/main.go

```diff
@@ -7,13 +7,15 @@ import (
 	"time"
 	"os"
 
+	"github.com/gluster/gluster-prometheus/gluster_exporter/conf"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 var (
 	port = flag.Int("port", 8080, "Exporter Port")
 	metricsPath = flag.String("metrics-path", "/metrics", "Metrics API Path")
 	peerid = flag.String("peerid", "", "Gluster Node's peer ID")
+	configfile = flag.String("config", "", "Collectors configuration file")
```

Comment:
Config is used as command line argument, if config file is not specified then all reports will be disabled.

Reply:
Thats true as per this change. Should be have some default collectors to be enabled always?